src/ncurses_ui.cc

```diff
@@ -610,13 +610,16 @@ Optional<Key> NCursesUI::get_next_key()
     if (not c)
         return {};
 
-    static constexpr auto convert = [](Codepoint c) -> Codepoint {
+    const cc_t erase = m_original_termios.c_cc[VERASE];
+    static auto convert = [erase](Codepoint c) -> Codepoint {
```

Comment:
This was static constexpr so that `parse_key` and `parse_csi` did not need to capture it, but now making it just static is weird as it means it will use the value of m_original_termios at first call, which should not change in the current way Kakoune works, but would break if we moved one day to say have the server handle multiple ncurses clients directly instead of having multiple client processes.

So this should probably be non-static and I guess we could make `parse_key` and `parse_csi` capture it by reference for now, until I find an idea I like better and refactor that code.